test.py

```diff
@@ -0,0 +1,51 @@
+import numpy as np
+import pandas as pd
+import io, csv, os
+import redis
+import pika
+
+def initialize(college_file, cols_file):
+	columnString = ""
+	collegeString = ""
+	with open(college_file) as csvfile:
+		reader = csv.reader(csvfile)
+		next(reader, None)
+		for row in reader:
+			collegeString += row[0] + "\n"
+
+	with open(cols_file) as txtfile:
+		line = txtfile.readline().strip()
+		line = line.split(',')
+		line.sort()
+		for i in line:
+			columnString += i + "\tnumerical\n"
+
+	return collegeString, columnString
+
+def testFeaturesCSV(column_string, r):
+	output_features = r.get('learning:college_features.csv')
+	#output_features_text = output_features.decode('utf-8').rstrip()
+	#print output_features_text
+	#print column_string
+	assert output_features == column_string, 'Error with features.txt' 
+
+
+def testCollegesCSV(college_string, r):
```

Comment:
camel case is unconventional for python, generally people go with underscores/lowercase